crates/orderbook-client/src/lib.rs

```diff
@@ -38,7 +38,7 @@ pub async fn subscribe_with_authentication(
     SplitSink<WebSocketStream<MaybeTlsStream<TcpStream>>, tungstenite::Message>,
     impl Stream<Item = Result<String, Error>> + Unpin,
 )> {
-    let signature = authenticate(create_sign_message());
+    let signature = authenticate(create_sign_message(b"Hello it's me Mario".to_vec()));
```

Comment:
Should we rather use a constant for this text somewhere?

Reply:
Sure why not.

Reply:
19b59d6c6a283d35a4f711098c66a31332fee1fb